src/OrleansAzureUtils/Hosting/AzureClient.cs

```diff
@@ -123,12 +140,12 @@ private static void InitializeImpl_FromFile(FileInfo configFile)
             InitializeImpl_FromConfig(config);
         }
 
-        private static string GetDeploymentId()
+        internal static string GetDeploymentId()
         {
             return GrainClient.TestOnlyNoConnect ? "FakeDeploymentId" : serviceRuntimeWrapper.DeploymentId;
```

Comment:
Out of curiosity, would there be a need to use "FakeDeploymentId" elsewhere? E.g. in some comparison operator?